src/style/themes/base/index.js

```diff
@@ -0,0 +1,17 @@
+import configureBase from './base_theme.config';
+import baseColors from '../../color-config';
+import atOpacity from '../../utils/at_opacity';
+import { mergeDeep } from '../../utils/merge_deep';
+import generatePalette from '../../palette';
+
+const palette = {
+  ...generatePalette(baseColors),
+  atOpacity
+};
+
+export default (configureTheme) => {
```

Comment:
can you separate the merge function from the theme? so you export the base theme here, and you export a merge function separately? Just thinking someone may want to use the base theme, or merge different themes together etc